instagram/src/components/CommentSection/NewCommentForm.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import './Style.css';
+
+const NewCommentForm = props => {
+  const cmtUsr = 'James'
+
+  return (
+  <div className="AddNewComment">
+    <form className="post-form" onSubmit={event => {
+      props.addNewComment(event, props.postUsr, cmtUsr, moment().format('MMMM Do YYYY, h:mm:ss a'));
```

Comment:
Good job on implementing stretch from previous projects (momentjs) and using it on your current project when it's not required.